src/test/java/net/sf/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -1205,16 +1204,15 @@ public void testCommandLineSinglePdf() throws IOException, TransformerException,
      * Test whether the command-line client can pick one of several entries from a bibtex file
      * @throws IOException
      * @throws TransformerException
-     *
      */
     @Test
-    @Ignore
     public void testCommandLineByKey() throws IOException, TransformerException {
 
         File tempBib = File.createTempFile("JabRef", ".bib");
         try (BufferedWriter fileWriter = Files.newBufferedWriter(tempBib.toPath(), StandardCharsets.UTF_8)) {
             fileWriter.write(t1BibtexString());
             fileWriter.write(t2BibtexString());
+            fileWriter.close();
 
             { // First try canh05
```

Comment:
Not your change, but why is this bracket opening in a new line? Can you fix the formatting in this method?